### PR TITLE
fixes the broken order flow

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes https://github.com/fossasia/open-event-frontend/issues/2994

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Currently, order flow is broken due to one of the recent merged PR.

### CODE REVIEW

File: app/api/orders.py [hunk 1]
```diff
@@ -287,10 +287,12 @@ def before_update_object(self, order, data, view_kwargs):
...
 
         elif current_user.id == order.user_id:
-            if order.status != 'pending':
+            print(order.status)
+            if order.status != 'initializing':
                 raise ForbiddenException({'pointer': ''},
-                                         "You cannot update a non-pending order")
+                                         "You cannot update a non-initialized order")
             else:
+                print('Hey----------------')
```
Comment (CosmicCoder96) [inline app/api/orders.py]: Remove this.

File: app/api/orders.py [hunk 2]
```diff
@@ -287,10 +287,12 @@ def before_update_object(self, order, data, view_kwargs):
                                                      "You cannot update the status of a cancelled order")
 
         elif current_user.id == order.user_id:
-            if order.status != 'pending':
+            print(order.status)
+            if order.status != 'initializing':
```
Comment (CosmicCoder96) [inline app/api/orders.py]: This is again wrong. Pending orders can be updated. Users can pay and complete them.

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 I've tried ordering a ticket -
 - paid and free tickets for logged in user
 - paid and free tickets for non-guest logged out user
 - paid and free tickets for guest user

However, when I tried to click on `Pay via card` from non-superadmin account, nothing happened and an error appeared on console - `Error: [ember-cli-stripe] Missing required `key` param`. Don't know if this issue was present earlier or not.

**Comment by CosmicCoder96** [comment]
It wasn’t present before. Maybe the PR which updated the dependencies
caused this. Either way please open an issue on FE to fix this. The library
might have changed after upgrade.

**Comment by CosmicCoder96** [comment]
Need to merge this urgently to find out other broken things.

**Pull request merged**